### DISCUSSION TITLE
Increase `traceFieldName` field length from 32 characters

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5943] [Feature Requests]
> Here i want to report a small suggestion to make the field traceFileNames (dnasequencingrun table) length 64 letters (currently it 32, which is may be not sufficient for larger file names).
> 
> For example, we have a standard names of our sequences about 34-40 letters, like this: 
> `1420_ITS_F_H04_08_2024-03-29-10-41-25.ab1`
> 
> ![image](https://github.com/specify/specify7/assets/37256050/b87a667d-9b05-49db-a8b8-e8adc5d3b8ee)

**Requested by:** Nina at Yugra State University on the [Speciforum](https://discourse.specifysoftware.org/t/tracefilename-field-length/1707)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/tracefilename-field-length/1707/2

**Comment by acwhite211** [comment]
This will be a good first schema change to do after #4358 is done 😀

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8190
